build

```diff
@@ -62,4 +62,5 @@ docker run --rm -e "RELEASE=${release}" -v "${package_temporary_dir}:/dist" "${i
 chown -R "${current_user}" "${package_temporary_dir}" &>/dev/null \
   || sudo chown -R "${current_user}" "${package_temporary_dir}" &>/dev/null
 # Move the APKs to the parent directory
-mv "${package_temporary_dir}"/apk/*.apk ../
+mkdir ../bin &>/dev/null || true
```

Comment:
```suggestion
mkdir -p ../bin &>/dev/null
```

Reply:
The upper dir always exists by definition, and -p won't avoid the failure condition.

Reply:
Just tried on my local Debian 9 box:
```sh
$ mkdir -p . ; echo $?
0
```
So my suggestion should work :)

Reply:
TIL thought i tried that...